src/Factory.php

```diff
@@ -229,6 +229,14 @@ private function normalizeAttribute($value)
             return $value->object();
         }
 
+        if ($value instanceof FactoryCollection) {
+            $this->afterPersist[] = function(Proxy $proxy) use ($value) {
```

Comment:
Simplify to `[$value, 'createFor']` for callback?

Got the following error: 

`TypeError: ReflectionFunction::__construct() expects parameter 1 to be string, array given`

From `Proxy::executeCallback()`